### PR TITLE
[TE][TensorIR] fix tensor attr in create_prim_func

### PR DESCRIPTION
**Pull request by Hzfengsy**
Convert `Tensor` into `Buffer` during `te.compute` attrs to enable TVMScript printing and parsing.

cc @wrongtest @junrushao1994

### CODE REVIEW

**Comment by junrushao1994** [comment]
@zxybazh and I observed the same issue for layout free placeholders

**Comment by junrushao1994** [comment]
@wrongtest would you like to take a look at the CI crash? Thanks a lot!

**Comment by wrongtest** [comment]
I am not able to reproduce crash of task_python_unittest.sh in local env. Is this a stable failure in CI?

**Comment by junrushao1994** [comment]
Weird...Unfortunately I'm on vacation and don't have a laptop in hand :-(

File: src/te/operation/create_primfunc.cc [hunk 1]
```diff
@@ -144,7 +145,30 @@ BlockRealize GenerateBlockFromTensor(const te::ComputeOp& compute_op, const te::
...
 
   // Step 6. Add script_parsing_detect_access attr for auto complete the whole IR.
-  Map<String, ObjectRef> annotations = compute_op->attrs;
+  Map<String, ObjectRef> annotations;
+  auto mutate_attr = [&info](const ObjectRef& value) -> ObjectRef {
+    if (value->IsInstance<te::TensorNode>()) {
+      const auto& tensor_value = Downcast<te::Tensor>(value);
+      auto it = info->tensor2buffers.find(tensor_value);
+      ICHECK(it != info->tensor2buffers.end());
+      return it->second;
```
Comment (junrushao1994) [inline src/te/operation/create_primfunc.cc:154]:
```suggestion
    if (const auto* tensor_value = value.as<te::TensorNode>()) {
      return info->tensor2buffers.at(GetRef<te::Tensor>(tensor_value));
```

**Comment by junrushao1994** [comment]
I did another round but a bit unsure where the issue could possible come from...Would you mind set the cmake build type to "Release" and try again?

**Comment by wrongtest** [comment]
I notice that the check is successful now. So the error is due to binding the const reference to a temp vallue? :) @Hzfengsy

**Comment by Hzfengsy** [comment]
I'm not sure. But just apply @junrushao1994 's suggestion

**Comment by junrushao1994** [comment]
okay to be more careful, I retriggered the CI to see if the problem is really gone.

**Review by junrushao1994 — APPROVED** [review]
LGTM

**Pull request merged**